packages/ketchup/src/components/kup-input-panel/kup-input-panel.tsx

```diff
@@ -623,15 +638,16 @@ export class KupInputPanel {
         cells: InputPanelCells,
         section: KupInputPanelLayoutSection,
         customLabelRender: boolean = false,
-        styleObj: GenericObject = {},
-        layout?: KupInputPanelLayout
+        styleObj: GenericObject = {}
     ) {
         const classObj = {
             'input-panel__section': !section.horizontal,
             'input-panel__horizontal-section': section.horizontal,
-            'input-panel__section-inline': layout?.horizontal,
+            'input-panel__section-inline': this.inputPanelPosition == 'INLINE',
         };
 
+        console.log(classObj);
```

Comment:
remove console.log.
also, new props should be added to the enum in declarations, otherwise getProps() won't work

Reply:
Fix both enumeration declarations and console.log